apps/omg_watcher_rpc/config/config.exs

```diff
@@ -15,7 +15,7 @@ config :omg_watcher_rpc, OMG.WatcherRPC.Web.Endpoint,
 config :omg_watcher_rpc, OMG.WatcherRPC.Tracer,
   service: :web,
   adapter: SpandexDatadog.Adapter,
-  disabled?: {:system, "METRICS", false},
+  disabled?: {:system, "DD_DISABLED", false, {String, :to_existing_atom}},
```

Comment:
I see this repeated a lot - isn't there an option to put this `config` entry once under `omg_status` and reuse?

Reply:
Sadly I'm not aware of an approach that would make this less confusing...